doc/static/downloads/build_acrn_ovmf.sh

```diff
@@ -0,0 +1,207 @@
+#
+# OUTPUT: ${your_working_directory}/acrn-edk2/Build/OvmfX64/DEBUG_GCC5/FV/OVMF.fd
+#
+# For more information, ./build_acrn_ovmf.sh -h
+#
+
+gop_bin_dir="./gop"
+docker_image_name="ubuntu:ovmf.16.04"
+proxy_conf="proxy.conf"
+
+if [ ! -x "$(command -v docker)" ]; then
+    echo "Please install Docker first:"
+    echo "If you are using Ubuntu, you can refer to: https://docs.docker.com/engine/install/ubuntu/"
+    exit
+fi
+
+if [ ! -d "${gop_bin_dir}" ]; then
+    mkdir gop
+    echo "Please copy IntelGopDriver.efi and Vbt.bin to ${gop_bin_dir}"
+    exit
+fi
+
+if [ ! -f "${gop_bin_dir}/IntelGopDriver.efi" ]; then
+    echo "Please copy IntelGopDriver.efi to ${gop_bin_dir}"
+    exit
+fi
+
+if [ ! -f "${gop_bin_dir}/Vbt.bin" ]; then
+    echo "Please copy Vbt.bin to ${gop_bin_dir}"
+    exit
+fi
+
+if [ ! -f "${proxy_conf}" ]; then
+    touch "${proxy_conf}"
+fi
+
+usage()
+{
+    echo "$0 [-i] [-s] [-h]"
+    echo "  -i:    Delete the existing Docker image ${docker_image_name} and re-create it"
+    echo "  -s:    Delete the existing acrn-edk2 source code and re-download/re-patch it"
+    echo "  -h:    Show this help"
+    exit
+}
+
+re_download=0
+re_create_image=0
+
+while getopts "his" opt
+do
+    case "${opt}" in
+        h)
+            usage
+            ;;
+        i)
+            re_create_image=1
+            ;;
+        s)
+            re_download=1
+            ;;
+        ?)
+            echo "{OPTARG}"
+            ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [[ "${re_create_image}" -eq 1 ]]; then
+    if [[ "$(docker images -q ${docker_image_name} 2> /dev/null)" != "" ]]; then
+        echo "===================================================================="
+        echo "Deleting the old Docker image ${docker_image_name}  ..."
+        echo "===================================================================="
+        docker image rm -f "${docker_image_name}"
+    fi
+fi
+
+if [[ "${re_download}" -eq 1 ]]; then
+    echo "===================================================================="
+    echo "Deleting the old acrn-edk2 source code ..."
+    echo "===================================================================="
+    sudo rm -rf acrn-edk2
+fi
+
+create_acrn_edk2_workspace()
+{
+    echo "===================================================================="
+    echo "Downloading & patching acrn_edk2 source code ..."
+    echo "===================================================================="
+
+    [ -d acrn-edk2 ] && sudo rm -rf acrn-edk2
+
+    git clone https://github.com/projectacrn/acrn-edk2.git
+    if [ $? -ne 0 ]; then
+        echo "git clone acrn-edk2 failed"
+        return 1
+    fi
+
+    cd acrn-edk2
+    git submodule update --init CryptoPkg/Library/OpensslLib/openssl
+    if [ $? -ne 0 ]; then
+        echo "git submodule acrn-edk2 failed"
+        return 1
+    fi
+
+    wget -q https://projectacrn.github.io/latest/_static/downloads/Use-the-default-vbt-released-with-GOP-driver.patch
```

Comment:
> 
> 
> This creates a dependency on the `/latest/` folder from an (older) tagged version of the release. This wget needs to fetch from the appropriate tagged release's documentation tree (/2.4/ for example). We should talk about how to make this work (maybe add a configuration variable with the release name (e.g., "latest" or "2.4" corresponding to the release version)? @gvancuts @NanlinXie any good ideas here?

No, I wanted to put a comment in there too but refrained from doing so since I did not have any better idea :-)

> 
> Here's one way I figured out that automatically figures this out by using `git describe` to find what tagged branch we're on and figures out the folder name in the documentation to use (either `latest` or the branch name without the leading `v`), but might need some robustness testing:
> 
> ```
> tag=`git describe --exact-match 2> /dev/null`
> folder=`[ -z "$tag" ] && echo latest || echo ${tag:1}`
> 
> wget -q https://projectacrn.github.io/${folder}/_static/downloads/.....
> ```

This could _nearly_ work but not quite because we're in the `acrn-edk2` clone at that point and we're always using the latest version... so this will invariably resolve to `latest`. **If** we introduce a mechanism to specify the ACRN version we want to use, then this could be used to **both** check out the corresponding version of `acrn-edk2` **and** the right version of the patch. This seems like the best option to me, i.e. add an argument to this script to build for a specific version, defaulting to `latest` if none is specified.

Reply:
I didn't notice we're in a clone of a different repo, sigh.  In that case, yes, we should add an argument for what version of the acrn-hypervisor (documentation) we should pull from or pull the script from a particular branch name in the git repo.  We're trying to solve an issue we have where scripts are referenced in tagged version documentation (e.g., 2.3) but aren't pointing to a script version specific to that tagged release (they're pointing to /latest).